src/Form/Admin/CommitteeMergeType.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+namespace AppBundle\Form\Admin;
+
+use AppBundle\Committee\CommitteeMergeCommand;
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+class CommitteeMergeType extends AbstractType
+{
+    public function buildForm(FormBuilderInterface $builder, array $options): void
+    {
+        $builder
+            ->add('sourceCommittee', CommitteeIdType::class)
```

Comment:
What about trying `ModelAutocompleteType` type?

Reply:
we discussed about that: we don't want an autocomplete, it could lead to confusions.
People who will use this tool must know the exact committee ID and will have to confirm everything.